lxqt-session/src/lxqtmodman.cpp

```diff
@@ -190,9 +188,11 @@ void LXQtModuleManager::startWm(LXQt::Settings *settings)
 {
     // if the WM is active do not run WM.
     // all window managers must set their name according to the spec
-    if (!QString::fromUtf8(NETRootInfo(QX11Info::connection(), NET::SupportingWMCheck).wmName()).isEmpty())
-    {
-        return;
+    if (auto x11NativeInterfce = qGuiApp->nativeInterface<QNativeInterface::QX11Application>()) {
+        if (!QString::fromUtf8(NETRootInfo(x11NativeInterfce->connection(), NET::SupportingWMCheck).wmName()).isEmpty())
```

Comment:
`11NativeInterfce->connection()` typo?

Reply:
No, it isn't a typo. Why should it be?

Reply:
The `Interfce` does look a bit odd fer sure.

Reply:
This is how it's done in Qt6. They got rid of `qt5-x11extras` and `QX11Info`.

Reply:
Was referring to the spelling `Interfce` instead of `Interface`.  : ]

Reply:
LOL! Why didn't you say it the first time? I thought you both meant _the code_. It's a known fact that the brain corrects such typos while reading ;)

Thanks! I'll correct them; I copy-pasted them from libfm-qt; so I'll correct them there too.

Reply:
Heh, yeah, my intention was to be very clear about it but realised it a bit too late that I really wasn't... sorry about that!